package.json

```diff
@@ -31,6 +31,7 @@
   },
   "dependencies": {
     "deep-equal": "^1.0.1",
+    "lodash": "^4.17.4",
```

Comment:
We want to avoid taking `lodash` as a dependency. It looks like you can use ES6 functions like `Array.isArray` in the places you're using lodash. Is there an alternative `_.groupBy` and `_.kebabCase` you could use? If not can you add [lodash.groupby](https://www.npmjs.com/package/lodash.groupby) and lodash.kebabcase as a dependency instead of the full lodash package?